src/lib/api-client.js

```diff
@@ -184,3 +202,14 @@ export function getModelsForNewItem() {
         hashHistory.push('/items/new');
     });
 }
+export function deleteItem(address){
+    return del('item', {
+        itemAddress: address
+    }).then(data => {
+        Dispatcher.handleAction('ITEMS_RECEIVED', data);
```

Comment:
1. It's better to defined two different actions if you know they're going to be consumed differently by stores. We will probably want to differentiate between a simple ITEMS_RECEIVED which only updates items, and an ITEM_DELETED which would also be useful for the toast store.
2. If Jordan's PR doesn't need to differentiate, it seems out of his scope. I don't think it's a blocking issue, and the action can be changed by the toast-related PR which needs to differentiate between the two.